src/integration.tests/integration.test.js

```diff
@@ -204,14 +204,52 @@ describe('Differencify', () => {
       .goto('http://example.com/')
       .waitFor(1000)
       .screenshot()
-      .toMatchSnapshot()
+      .toMatchSnapshot((resultDetail) => {
```

Comment:
Probably better to write a new test instead of updating the existing one. These tests are good examples for differencify usage